src/blister.c

```diff
@@ -219,7 +219,7 @@ static Obj DoCopyBlist(Obj list, Int mut)
 
     /* return the copy                                                     */
     return copy;
-  
+
```

Comment:
I've been told before on numerous occasions by @ChrisJefferson that pure white spaces changes are annoying and should be avoided. It seems he is fine with that here, and I'd normally not care about it myself, *but* here they really conflict with a WIP PR of mine, so it'd be nice if these could be removed from this PR. I'd be happy to perform the required changes with your permission (by pushing a commit with them to this PR), as this is trivial to do with a little git magic (using `git reset` plus `git add -p` resp. `git checkout -p` etc.). Of course you can also do them yourself.

Reply:
Ooops, this was a mistake. I'm happy to remove it.

Reply:
Thank you! Please post a comment once you are done updating this, so we'll see it -- as you know, unfortunately (or fortunately? ;-)it'd be nice if at least one could choose...) GitHub does not send notifications when a PR is updated.

Reply:
Weirdly I didn't see them when I reviewed it, maybe they got added later. They do generally annoy me, just because they make reviewing harder

Reply:
Perhaps you have "Hide whitespace changes" selected in the diff settings when reviewing a PR (and if you're not using this option, maybe using it will help you get less annoyed!)